src/dashboard/src/hooks/filter/useFilteredFileSystemItems.ts

```diff
@@ -4,11 +4,11 @@ import { IFileSystemItemFilter, IFileSystemItemModel, useApiFileSystemItems } fr
 
 export const useFilteredFileSystemItems = () => {
   const { find } = useApiFileSystemItems();
+  const isLoading = useFilteredStore((state) => state.loadingFileSystemItems);
+  const setIsLoading = useFilteredStore((state) => state.setLoadingFileSystemItems);
   const fileSystemItems = useFilteredStore((state) => state.fileSystemItems);
   const setFileSystemItems = useFilteredStore((state) => state.setFileSystemItems);
 
-  const [isLoading, setIsLoading] = React.useState(false);
```

Comment:
Look hook state was causing the issue.  Every different hook would have it's own state.  As such the hook for the filter was not the same state variable as the one used on the dashboard.  Loading state has been moved to the zustand store.